src/api/public/apidocs-new/paths/distributions.yaml

```diff
@@ -0,0 +1,117 @@
+get:
+  summary: List all base distributions hosted on this OBS instance.
+  description: |
+    Get a list of all base distributions hosted on this OBS instace.
+    This will not list Distributions hosted remotely.
```

Comment:
```suggestion
    This will not list distributions that are available via [interconnect](https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.concepts.html#id-1.5.10.3.5).
```